### PR TITLE
feat: Update the design of the audit logs page

### PR DESCRIPTION
**Pull request by pranavrajs**
This is continuation of the design update, updates the design for audit logs listing page.

| Light | Dark |
| -- | -- |
| Screenshot 2024-08-06 at 5 34 09 PM | Screenshot 2024-08-06 at 5 33 52 PM |

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue [hunk 1]
```diff
@@ -1,137 +1,126 @@
...
 import TableFooter from 'dashboard/components/widgets/TableFooter.vue';
+import BaseSettingsHeader from '../components/BaseSettingsHeader.vue';
 import {
   generateTranslationPayload,
   generateLogActionKey,
 } from 'dashboard/helper/auditlogHelper';
+import { computed, onMounted, watch } from 'vue';
+import { useI18n } from 'dashboard/composables/useI18n';
+import { useRoute } from 'dashboard/composables/route';
+import router from '../../..';
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue:14]: @pranavrajs Can we use `useRouter` from `composables/route`?

**Review by iamsivin — APPROVED** [review]
LGTM

File: app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue [hunk 2]
```diff
@@ -1,137 +1,126 @@
...
+          </tr>
+        </tbody>
+      </table>
+      <TableFooter
+        :current-page="Number(meta.currentPage)"
+        :total-count="meta.totalEntries"
+        :page-size="meta.perPage"
+        class="border-slate-50 dark:border-slate-800 border-t !px-0 py-4"
+        @pageChange="onPageChange"
+      />
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue:123]: Can we hide the table pagination while the audit log is fetching?

Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue]: Done.

Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue]: Great catch!

File: app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue]: Done.

**Comment by pranavrajs** [comment]
@iamsivin Review comments are fixed. Please take a look.

**Review by iamsivin — APPROVED** [review]

**Pull request merged**